src/vtcomposite.cpp

```diff
@@ -275,14 +275,21 @@ NAN_METHOD(composite)
 
     std::unique_ptr<BatonType> baton_data = std::make_unique<BatonType>(num_tiles);
 
+    v8::Isolate *isolate = v8::Isolate::GetCurrent();
```

Comment:
We should use NAN instead of using `v8::Isolate` directly.

Reply:
This was just a quick fix to get travis green again. Agree, using `Nan` wrappers is a better style but in this case it's functionally the same as far as I can see. Fix is on its way.

Reply:
https://github.com/mapbox/vtcomposite/commit/faf52cb6a7122f5f0b820307ee7c70a45228c884